src/client/client.js

```diff
@@ -1,5 +1,6 @@
 import {render} from 'react-dom';
 import React from 'react';
+import { AppContainer } from 'react-hot-loader';
```

Comment:
this breaks linting, right? let's get to 100% lint-free & get it plugged into circleCI